src/api/app/policies/user_policy.rb

```diff
@@ -6,4 +6,8 @@ def update?
   def show?
     user.can_modify_user?(record)
   end
+
+  def see_rss?
+    user.login == record.login && record.rss_token
```

Comment:
@saraycp Am I correctly understood, that it may be changed like this?

```suggestion
    Token::RssPolicy.new(user, record).create?
```